src/MutableMap/MutableMap.dfy

```diff
@@ -0,0 +1,120 @@
+/*******************************************************************************
+*  Copyright by the contributors to the Dafny Project
+*  SPDX-License-Identifier: MIT
+*******************************************************************************/
+
+// RUN: %verify "%s"
+
+include "../../src/Wrappers.dfy"
+
+/**
+  *  Implements mutable maps by interfacing with external code, e.g. "MutableMap.java".
+  */
+
+module {:extern "DafnyLibraries"} {:options "-functionSyntax:4"} DafnyLibraries {
+  import opened Wrappers
+
+  /**
+  *  NOTE: Only here because of #2500; once resolved import "../../examples/MutableMap/
+  *  MutableMapTrait.dfy".
```

Comment:
nit for later: it seems useful to have the trait for consumers too, so my preference would be to remove the example copy instead. But this is fine for now.